internal/build/docker_test.go

```diff
@@ -334,6 +334,22 @@ type pathContent struct {
 	contents string
 }
 
+func (f *testFixture) startContainerWithOutput(ctx context.Context, ref string, cmd *tiltd.Cmd) string {
+	cId, err := f.b.startContainer(ctx, ref, cmd)
+	if err != nil {
+		f.t.Fatal(err)
+	}
+
+	out, err := f.dcli.ContainerLogs(ctx, cId, types.ContainerLogsOptions{ShowStdout: true, ShowStderr: true})
+	if err != nil {
+		f.t.Fatal(err)
+	}
+
+	output := &strings.Builder{}
+	io.Copy(output, out)
```

Comment:
ioutil.ReadAll would be a better choice here

Reply:
fyi @jazzdan -- i didn't know about this thing, did you?!

Reply:
Nope, TIL!